src/Tribe/Editor.php

```diff
@@ -543,9 +550,19 @@ public function deregister_scripts() {
 	 *
 	 * @since 4.7
 	 *
+	 * @param array<array<string|string>> $categories An array of categories each an array
+	 *                                                in the format property => value.
+	 * @param WP_Post                     $post       The post object we're editing.
+	 *
 	 * @return array
 	 */
 	public function block_categories( $categories, $post ) {
+		// Handle where someone is using this outside of this object
+		global $wp_version;
+		if ( class_exists( 'WP_Block_Editor_Context' ) || version_compare( $wp_version, '5.8', '>=' ) ) {
```

Comment:
As above: should we drop the `class_exists` check here and just run the `version_compare`?

Reply:
I would drop the `class_exists` check completely, and use `$context instanceof WP_Block_Editor_Context` right in the method that needs it.